src/core/typography/H1.test.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import 'jest-styled-components'
+import styled from 'styled-components'
```

Comment:
I'd be curious if tests still pass without this line, but could be wrong 🤔

Reply:
I was trying something and forgot to take it out! Oops. I'll remove it! Thanks for catching.